core/node/node_framework/src/implementations/layers/web3_api/server/sealed_l2_block.rs

```diff
@@ -0,0 +1,50 @@
+use std::time::Duration;
+
+use zksync_dal::{Core, CoreDal};
+use zksync_db_connection::connection_pool::ConnectionPool;
+use zksync_node_api_server::web3::state::SealedL2BlockNumber;
+
+use crate::{StopReceiver, Task, TaskId};
+
+#[derive(Debug)]
+pub struct SealedL2BlockUpdaterTask {
```

Comment:
Nit: AFAIU, the idea with the `node_framework` crate is to eventually invert its dependency on node components (so that components depend on the framework). So, it makes slightly more sense to define tasks in the component crates so they don't need to be moved there in the future. This is a very minor thing however.

Reply:
Do you mean to define tasks in component crate but implement in `node_framework`? If so then I find it confusing (for now) and would prefer to do it later when we start inverting globally